static/js/drafts.js

```diff
@@ -392,10 +411,40 @@ export function launch() {
         return sorted_formatted_drafts;
     }
 
-    function render_widgets(drafts) {
+    function render_widgets(narrow_drafts, other_drafts) {
```

Comment:
It is probably worth a block comment here explaining the algorithm; that might obviate the need for the `Compose box headers get priority.` comment below; I'm not sure.